src/generators/web.cr

```diff
@@ -73,6 +74,10 @@ class LuckyCli::Generators::Web
     FileUtils.rm_r("#{project_dir}/src")
   end
 
+  private def remove_generatred_spec_files
```

Comment:
Small typo in "generated"

Reply:
Nevermind, I realized I can do this pretty easily in the GitHub UI after merging.

Reply:
Awesome, thanks 👍